src/components/_LuxLibraryContactInfo.vue

```diff
@@ -58,7 +58,6 @@ export default {
     background-color: var(--color-gray-100);
     a {
       color: var(--color-white);
-      &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-dark);
}